.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: rust
-sudo: false
+cache: cargo
```

Comment:
are you sure that's a good idea? My understanding was that the cache is huge in size and the costs of keeping it are not always justified.

Reply:
I'm not sure. I hadn't heard that, I had just assumed it was better from experience with other languages. It looks like there is a mix of projects that use `cache: cargo`, some that cache only `$HOME/.cargo/` and some that don't use caching at all, I'm not sure which is better.

Reply:
The problem with caching is cargo never cleans up old files built from old rustc versions or old dependencies.